presto-common/src/main/java/com/facebook/presto/common/Page.java

```diff
@@ -418,12 +422,17 @@ public Page dropColumn(int channelIndex)
 
     private long updateRetainedSize()
     {
-        long retainedSizeInBytes = INSTANCE_SIZE + sizeOf(blocks);
+        AtomicLong retainedSizeInBytes = new AtomicLong(INSTANCE_SIZE + sizeOf(blocks));
```

Comment:
Are we using `AtomicLong` just for mutability?

Reply:
I think it's for lambda reference

Reply:
yes, its for lambda reference